ecommerce/courses/models.py

```diff
@@ -129,6 +129,10 @@ def enrollment_code_product(self):
 
     def get_course_seat_name(self, certificate_type, id_verification_required):
         """ Returns the name for a course seat. """
+        if getattr(settings, 'ENABLE_NEW_COURSE_SEAT_NAME', False):
```

Comment:
We should use a setting name to suggest this is a custom change.
```suggestion
        if getattr(settings, 'ENABLE_CUSTOM_COURSE_SEAT_NAME', False):
```